Source/Fuse.Controls.Navigation/NavigationControl.Pages.uno

```diff
@@ -6,43 +6,43 @@ namespace Fuse.Controls
 	public partial class NavigationControl
 	{
 		//TODO: Change to IObservableArray once Model feature is merged
-		IArray _pages;
-		Uno.IDisposable _pagesSubscription;
+		IArray _pageHistory;
+		Uno.IDisposable _pageHistorySubscription;
 		/**
 			Pages is a stack of pages that controls the local history for a NavigationControl.
 			
 			It should be bound to a JavaScript observable array. The highest index page will always be the active page for the control. As pages are added/removed from this array the navigation state will change.
 			
 			The items in the array are objects, either explicitly created or via the Model feature. They should contain the the `$path` property which specifies the path to use. The object itself will be added to the data context for the page, allowing lookups from within the object.
 		*/
-		public IArray Pages
+		public IArray PageHistory
```

Comment:
This is a public API change, and should be mentioned in the changelog, at least. But perhaps we should provide some backwards compatibility here also?

Reply:
This wasn't released yet. `Pages` is only on master.